server/service/src/backend_plugin/boajs/methods/get_plugin_data.rs

```diff
@@ -0,0 +1,35 @@
+use boa_engine::*;
+use repository::{PluginDataFilter, PluginDataRepository, PluginDataRow};
+
+use crate::backend_plugin::{boajs::utils::*, plugin_provider::PluginContext};
+
+pub(crate) fn bind_method(context: &mut Context) -> Result<(), JsError> {
+    context.register_global_callable(
+        JsString::from("get_plugin_data"),
+        0,
+        NativeFunction::from_copy_closure(move |_, args, mut ctx| {
+            // TODO Is this actually safe ? (need to check reference counts after plugin has run)
```

Comment:
Would like to know about this before we merge..?

Reply:
Please see other comments and this commit: [b5483e5](https://github.com/msupply-foundation/open-msupply/pull/6684/commits/b5483e59f16b07d6a5c504410f6b20a5cf23a2cd)